src/renderer/components/layout/topbar.module.css

```diff
@@ -42,4 +43,7 @@
   align-items: center;
   display: flex;
   height: 100%;
+}
+
+.extensionItem {
```

Comment:
Is this needed?

Reply:
Was thinking about it as a placeholder for future styling. Removing for now.